### PR TITLE
publish custodian wallets

### PR DESCRIPTION
**Pull request by kziemianek**
* publish enclave's bitcoin and ethereum public keys in parachain bitacross pallet.

Keys are always published during the startup - even if they are no freshly generated but I assume it's enough for first version.

### CODE REVIEW

File: pallets/bitacross/src/custodial_wallet.rs [hunk 1]
```diff
@@ -15,14 +15,15 @@
 // along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
 
 use codec::{Decode, Encode};
-use core_primitives::{Address20, Address33};
 use scale_info::TypeInfo;
 
+pub type PubKey = [u8; 33];
```
Comment (Kailai-Wang) [inline pallets/bitacross/src/custodial_wallet.rs:20]: Did you try it?

Are `ethereum_key.public_bytes` and `bitcoin_key.public_bytes` guaranteed to be convertible to this type?

Comment (kziemianek) [reply pallets/bitacross/src/custodial_wallet.rs:20]: Yes, it's 
```
pub type CompressedPoint = GenericArray<u8, U33>;
```

Comment (Kailai-Wang) [reply pallets/bitacross/src/custodial_wallet.rs:20]: Ah I see, so it's the pubkey, not address

**Review by Kailai-Wang — APPROVED** [review]

**Review by felixfaisal — APPROVED** [review]
It looks good, yeah not sure if the bitacross colleagues need pubkey or address

**Pull request merged**